vllm/model_executor/models/gemma.py

```diff
@@ -248,15 +236,17 @@ def forward(
         # Normalize the embedding by sqrt(hidden_size)
         hidden_states = hidden_states * (self.config.hidden_size**0.5)
```

Comment:
we could also save some memory by doing
```suggestion
        hidden_states *= (self.config.hidden_size**0.5)
```

Reply:
Fixed!